apps/explorer/src/components/module/module-functions-interaction/ModuleFunction.tsx

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Disclosure } from '@headlessui/react';
+
+import type { SuiMoveNormalizedFunction } from '@mysten/sui.js';
+import type { ObjectId } from '@mysten/sui.js/src';
+
+import { ReactComponent as ChevronDownIcon } from '~/assets/SVGIcons/16px/chevron-down.svg';
+
+export type ModuleFunctionProps = {
+    packageId: ObjectId;
+    moduleName: string;
+    functionName: string;
+    functionDetails: SuiMoveNormalizedFunction;
+    defaultOpen?: boolean;
+};
+export function ModuleFunction({
+    defaultOpen,
+    functionName,
+    functionDetails,
+}: ModuleFunctionProps) {
+    return (
+        <div className="rounded-lg bg-sui-grey-40">
```

Comment:
Seems like a good thing to pull out into a base UI component

Reply:
Good catch I will move it there

Reply:
done